site/src/job_queue.rs

```diff
@@ -111,8 +107,8 @@ async fn create_benchmark_request_releases(
     // TODO; delete at some point in the future
     let cutoff: chrono::DateTime<Utc> = chrono::DateTime::from_str("2025-06-01T00:00:00.000Z")?;
 
-    for release_string in releases.lines() {
-        if let Some((name, date_time)) = parse_release_string(release_string)? {
+    for release_string in releases.lines().rev().take(20) {
```

Comment:
This is a bit different than we need - here you load the last 20 lines, but we need the last 20 release artifacts (imagine e.g. that we had 20 lines with mostly nightly releases, then we'd take only a few actual release artifacts. Check how the `parse_published...` function was used, we can just copy paste it.

Reply:
069f182c3f8a60eea7ddaf10df348db241bba761 I've changed it :+1: